src/main/resources/templates/src/main/java/MainVerticle.java.ftl

```diff
@@ -1,12 +1,20 @@
 package ${packageName};
 
 import io.vertx.core.AbstractVerticle;
+<#if vertxVersion?index_of("3") != 0>
+import io.vertx.core.Promise;
+<#else>
 import io.vertx.core.Future;
+</#if>
 
 public class MainVerticle extends AbstractVerticle {
 
   @Override
+  <#if vertxVersion?index_of("3") != 0>
+  public void start(Promise<Void> startFuture) throws Exception {
```

Comment:
Can you rename the parameter to `startPromise`?

Reply:
ok